Formula/kubeaudit.rb

```diff
@@ -0,0 +1,20 @@
+class Kubeaudit < Formula
+  desc "Helps audit your Kubernetes clusters against common security controls"
+  homepage "https://github.com/Shopify/kubeaudit"
+  url "https://github.com/Shopify/kubeaudit/archive/v0.6.0.tar.gz"
+  sha256 "fefd2d8255c59ecb94e326b7ea2a70e12005ac1aab4b1bcaca9dd86cc424679d"
+  head "https://github.com/Shopify/kubeaudit.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath/"src"
+
+    system "go", "build", "-o", "kubeaudit"
+    bin.install "kubeaudit"
+  end
+
+  test do
+    system "#{bin}/kubeaudit", "version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Sure thing. Just updated the test to try and run it with invalid credentials.